README.md

```diff
@@ -19,20 +19,29 @@ Currently supported platforms are Windows and Linux, x86_64 only.
 ### Required for Windows:
 * `python` installed
 * `cmake`
-* Visual Studio compilers, version depends on which Python version you want to target, see [correct MSVC version](https://wiki.python.org/moin/WindowsCompilers)
+* Visual Studio with release year not older than 2015 and Visual Studio compiler, which version depends on which Python version you want to target, see [correct MSVC version](https://wiki.python.org/moin/WindowsCompilers)
```

Comment:
why change? I don't see added value by this change...

Reply:
This change is confusing and not necessary, change was canceled in new commit.